### ISSUE TITLE
Unicode Greek letters don't parse

### ISSUE THREAD

**Issue by bengolds** [davidedc/Algebrite#30]
Algebrite chokes on parsing any expression with a unicode greek letter. EG: 3*γ

**Comment by bengolds** [comment]
Simple fix [here](https://github.com/bengolds/Algebrite/commit/ce9b8b91c647733df06cf7b1cc9a58cf5db28e40). May not be what you want, but it does work for now.

**Comment by davidedc** [comment]
yes I shall include it, but I'd also need to show that off in an example in the home page... also there should be appropriate latex support for those chars
